### PR TITLE
fix(TDI-46131): Missing jar for snowflake

### PR DESCRIPTION
**Pull request by olhava**
* aligned dependencies versions in snowflake with other TCOMPv0

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/808/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/808/)

File: components/components-snowflake/components-snowflake-runtime/pom.xml [hunk 1]
```diff
@@ -105,15 +105,13 @@
         </dependency>
         <!-- API -->
         <dependency>
-            <groupId>jakarta.xml.bind</groupId>
-            <artifactId>jakarta.xml.bind-api</artifactId>
-            <version>2.3.2</version>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
```
Comment (ypiel-talend) [inline components/components-snowflake/components-snowflake-runtime/pom.xml:109]: Where is defined the version of thos two libs ?

Comment (olhava) [reply components/components-snowflake/components-snowflake-runtime/pom.xml:109]: The versions are common for all the components and are defined in components-bom:
https://github.com/Talend/components/blob/8a4b767265fffcbd46dce64ba57335e7cf3f83ed/components-parent/components-bom/pom.xml#L590

**Review by ypiel-talend — APPROVED** [review]
LGTM

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/809/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/809/)

**Pull request merged**